src/components/tables/BaseTable.tsx

```diff
@@ -16,6 +15,7 @@ import {
 import type { TableStructure } from "./types";
 
 type Props<T> = {
+  actionItems?: JSX.Element;
```

Comment:
@0tuedon you made a similar change as part of #289, what do you think about this in comparison with your change?

Reply:
It does make sense to be able to customize it fully and add. I think the issue with mine was having the functions elsewhere (handleArchive) so this works.